validator/src/admin_rpc_service.rs

```diff
@@ -146,6 +152,14 @@ pub trait AdminRpc {
     #[rpc(meta, name = "exit")]
     fn exit(&self, meta: Self::Metadata) -> Result<()>;
 
+    #[rpc(meta, name = "reloadPlugin")]
+    fn reload_plugin(
+        &self,
+        meta: Self::Metadata,
+        libpath: String,
```

Comment:
We should only requires the config_file -- we have already documented the config file must have the "libpath" in its json content.

Reply:
the user might have moved the geyser plugin.

Reply:
marking as resolved as rpc methods only require config file